packages/utils/src/async.ts

```diff
@@ -1,5 +1,10 @@
 const TAG = 'utils/src/async'
 
+/** Wait for a number of milliseconds */
+export function wait(ms: number): Promise<void> {
```

Comment:
We technically have a npm package for it `sleep-promise`, and we keep redefining it everywhere. Maybe this is a good place but definitely would like to stream line. Also we have traditionally called this function `sleep` which might be good to do here as well

Reply:
renamed to sleep.

I wouldn't recommend adding a package for something like this.